### PR TITLE
Allow the use of the eyetracking component on nested gameobjects

### PR DESCRIPTION
**Pull request by peader**
## Overview
Allow the user to define a box collider on the parent object of the object referencing the eyetracking target script and still have it recognized as a valid "lookateyetarget"

## Additional Notes
I noticed the updatetesticons utility method is currently broken so I fixed that as well.

### CODE REVIEW

File: Assets/MRTK/SDK/Features/Input/Handlers/EyeTrackingTarget.cs [hunk 1]
```diff
@@ -260,8 +260,9 @@ private void UpdateHitTarget()
 
                     if (isHit)
                     {
-                        LookedAtTarget = hitInfo.collider.gameObject;
-                        LookedAtEyeTarget = LookedAtTarget.GetComponent<EyeTrackingTarget>();
+                        LookedAtEyeTarget = hitInfo.collider.transform.GetComponent<EyeTrackingTarget>() ??
+                        hitInfo.collider.transform.GetComponentInChildren<EyeTrackingTarget>();
+                        LookedAtTarget = LookedAtEyeTarget?.gameObject;
                         LookedAtPoint = hitInfo.point;
```
Comment (vaoliva) [inline Assets/MRTK/SDK/Features/Input/Handlers/EyeTrackingTarget.cs:266]: Hi @peader , Thanks for contributing to MRTK and also adding a test for your proposal.
I am thinking how could we handle a scenario where other hit parent children also have the EyeTrackingTarget component attached to?

Comment (peader) [reply Assets/MRTK/SDK/Features/Input/Handlers/EyeTrackingTarget.cs:266]: Hi @vaoliva , could you give me an example use case for such a feature?

**Comment by RogPodge** [comment]
@peader it's been a bit since we had a chance to last look at this PR. Do you have time to double check the failing tests/clean up the weird meta-file changes? Otherwise, we can take over this PR and have it on the backlog.

**Comment by RogPodge** [comment]
Closing this issue, since being able to activate nested game objects via parents doesn't quite work for our interaction model (ie, if two children have are eyetracking targets, how do we know which to set as the LookAtTarget?)

the tests and other QoL are being ingested in this PR #9704

**Pull request closed without merging**